### ISSUE TITLE
MPLE warning "glm.fit: fitted probabilities numerically 0 or 1 occurred"

### ISSUE THREAD

**Issue by krivit** [statnet/ergm#518]
We currently propagate this warning to the user, but is there any point it doing it? What is the user supposed to do or not do based on it? @handcock , @drh20drh20 , I think you put it in place.

**Comment by krivit** [comment]
Don't we already do detection of directions of recession with the code from Christian?

**Comment by handcock** [comment]
I believe it is produced by glm and can be removed. I left it in originally as I thought it might prove helpful in some circumstances but has not been, to my knowledge.

**Comment by mbojan** [comment]
I think I ran across this warning some time ago when fitting to a vector target statistics. I can't find it now what data and model these were. My uneducated guess at the time was that my targets imply to many ties for some subgroups to hold (thus some GLM on the way implied P=1...). However I just tried to cook-up a similar simplified scenario (such as the one below), but could not generate the warning. When is it supposed to appear? Perhaps it is not wise to suppress it, but rather to turn it into something more informative?

```r
library(ergm)
net0 <- network.initialize(24, directed = FALSE)
net0 %v% "group" <- rep(1:2, each = 12)
summary(net0 ~ mm(~ group, levels = NULL, levels2 = NULL))
fit <- ergm(
  net0 ~ mm(~ group, levels = NULL, levels2 = NULL),
  target.stats = c(7, 0, 77) # group 2 is complete for 66 ties, so 77 is too many
)
```
